Extensions/TextObject/textruntimeobject.ts

```diff
@@ -56,7 +56,8 @@ namespace gdjs {
     opacity: float = 255;
     _textAlign: string = 'left';
     _wrapping: boolean = false;
-    _wrappingWidth: float = 1;
+    // A wrapping of 1 makes games crash on Firefox
+    _wrappingWidth: float = 100;
```

Comment:
I'm not sure what a wrapping represents, 100 seems a lot or does it not matter much?

Reply:
It's the width of a text column. I don't think there is a good default value, but 1 is harmful.